added SECURITY.md (#29)
department-of-veterans-affairs/va.gov-platform-architecture: SECURITY.md

# Security Policy

The Department of Veterans Affairs participates in a Responsible Vulnerability Disclosure Program through Bugcrowd. Please report any security-related issues using our [Bugcrowd program](https://bugcrowd.com/va-vdp).

If your Bugcrowd submission is related to an open GH issue, please include that issue ID in the description field of your disclosure. 

> But, please do not open new security-related issues. If you are a developer and have concerns, you can reach out on our internal Slack channel #platform-security.

Reviewer: Can we link to the slack workspace?

Author: Added.

> But, please do not open new security-related issues. If you are a developer and have concerns, you can reach out on our internal Slack channel [#platform-security](https://dsva.slack.com/archives/C01CJV0L9PS).

Reviewer 2: ```suggestion
Please do not create security related issues in Github. If you are a developer and have concerns, you can reach out on our internal Slack channel [#platform-security](https://dsva.slack.com/archives/C01CJV0L9PS).
```
